src/Atom/AtomConverter.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Xml;
+
+namespace Microsoft.SyndicationFeed.Atom
+{
+    static class AtomConverter
+    {
+        public static bool TryParseValue<T>(string value, out T result)
```

Comment:
This is an exact copy of Converter.TryParseValue<T>(string value, out T result) so it isn't needed right? Converter can just be called directly.

Reply:
In my code changes, Converter was deleted and replaced with separate RSSConverter and AtomConverter static classes.  RSSConverter and AtomConverter do have identical TryParseValue functions.

I deleted Converter because the date serialization formats need to be different for RSS and Atom, hence separate classes.